generate/booklet.md

```diff
@@ -65,12 +65,12 @@ the sheet size of the paper you will be printing on, and the method you will be
 
 The n-up value will be how many pages of your booklet will fit on one side of the sheet you will be printing on. The following options are available:
 
-| n-up | portrait            | landscape           | multifolio |
```

Comment:
multifolio is now supported for all Nup booklets (forgot to update this earlier)